integrated_channels/sap_success_factors/models.py

```diff
@@ -93,9 +123,35 @@ class LearnerDataTransmissionAudit(models.Model):
     error_message = models.TextField(blank=True)
     created = models.DateTimeField(auto_now_add=True)
 
+    epoch = datetime(1970, 1, 1, tzinfo=timezone.utc)
```

Comment:
Again, @haikuginger 's https://github.com/edx/edx-enterprise/pull/63 helps us here.. then I can use his [`EPOCH`](https://github.com/edx/edx-enterprise/pull/63/files#diff-3727947e8e9274e6b3ed166f76f07d7cR15) and [`parse_datetime_to_epoch`](https://github.com/edx/edx-enterprise/pull/63/files#diff-3727947e8e9274e6b3ed166f76f07d7cR81)